src/classes/mech/MechEquipment.ts

```diff
@@ -61,8 +61,10 @@ abstract class MechEquipment extends LicensedItem {
       this.IsOrdnance = data.tags.some(x => x.id === 'tg_ordnance')
       this.CanSetDamage = data.tags.some(x => x.id === 'tg_set_damage_type')
       this.CanSetUses = data.tags.some(x => x.id === 'tg_set_max_uses')
+      this.max_use_override = this.CanSetUses ? 0 : null
     } else {
       this._max_uses = 0
+      this.max_use_override = null
```

Comment:
If max uses can be overridden, set the override to 0 by default (and distinguish between `null` and 0).  Assume that any equipment that has the `tg_set_max_uses` will ultimately require manual input (e.g. Prototype Weapon rolling for uses), and won't need a default (please check me on this, not sure if this should always be the case).